portal/worker/src/caching.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { resolveAndFetchPage } from "@lib/page_fetching";
+import { SuiClient, getFullnodeUrl } from "@mysten/sui/client";
+import { NETWORK } from "@lib/constants";
+import { DomainDetails } from "@lib/types";
+
+const CACHE_NAME = "walrus-sites-cache";
+// TODO - move it to .env
```

Comment:
I thought this over, and since the service worker is a one-time install, we should not be changing the `.env` arbitrarily like that without this change showing up somewhere. 

So let's keep it inside the code so that it passes through a PR whenever we need to change something like this.

Reply:
makes sense. Then remove the todo :)